client/my-sites/sharing/connections/service.jsx

```diff
@@ -364,10 +374,7 @@ export class SharingService extends Component {
 
 		const header = (
 			<div>
-				<SocialLogo
-					icon={ replace( this.props.service.ID, '_', '-' ) }
-					size={ 48 }
-					className="sharing-service__logo" />
+				{ this.renderLogo() }
```

Comment:
It allows us to override `renderLogo()` in the `GooglePhotos` class to provide a custom logo - this is the bit mentioned in the PR text about SocialLogos not supporting colour logos.

Code for that is here: https://github.com/Automattic/wp-calypso/pull/13711/files#diff-c659503b7917c8473d70728224788426R63